src/morphology.cpp

```diff
@@ -123,7 +123,7 @@ const std::vector<Section> Morphology::sections() const
     std::vector<Section> sections_;
     auto count = _properties->get<morphio::Property::Section>().size();
     sections_.reserve(count);
-    for (uint i = 0; i < count; ++i) {
+    for (unsigned int i = 0; i < count; ++i) {
```

Comment:
is count size_t?  if so, why not use that?

Reply:
Because then you need to cast in back to unsigned int when you use `i` in the Section constructor.